mousestyles/classification/classification.py

```diff
@@ -0,0 +1,127 @@
+from sklearn.grid_search import GridSearchCV
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.preprocessing import StandardScaler
+
+import numpy as np
+import pandas as pd
+
+
+def prep_data(strain, features, rseed=222):
+    """
+    Returns a list of 4: [train_y, train_x, test_y, test_x]
+        train_y: list of strain labels in train data sets,
+        train_x: list of features in train data sets,
+        test_y: list of strain labels in test data sets,
+        test_x: list of features in train data sets
+    Parameters
+    ----------
+    strain: ndarray, shape of (1921,)
+            classification labels
+    features: ndarray, shape of (1921, 99)
```

Comment:
Doesn't have to be in this shape. It's whatever the user wants to put in. Also, we might try using subset of features, since 99 features is a lot, especially if the user input the day-to-mouse-average data, which only have 170 observations.